docs/source/command_line.rst

```diff
@@ -344,10 +344,10 @@ Here are some more useful flags:
 .. _config-file-flag:
 
 - ``--config-file CONFIG_FILE`` causes configuration settings to be
-  read from the given file.  By default settings are read from ``mypy.ini``
```

Comment:
Please put the double spaces after periods back. They don't affect how the HTML is rendered, but they improve the quality of paragraph reflowing in Emacs.